### DISCUSSION TITLE
bloodsamplevial cannot be swung at 0 condition

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14459] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The traitor item `bloodsamplevial` cannot be swung as a melee weapon since it spawns at 0 condition and is considered "broken".

You have to open the medical UI and use it, which will increase its condition to 10 and then allow swinging it like a melee weapon

Just spawn it at 1% condition?

![image](https://github.com/user-attachments/assets/35bcf511-12dc-4a2e-94a5-bc3f4c912682)

### Reproduction steps

spawn `bloodsamplevial` at 0 condition in sub editor (for some reason spawning it in game via `spawnitem bloodsamplevial cursor` doesnt trigger onspawn)

try to swing it

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/fe1ccc31e6049df1ebc7cfb3d8f8a66308c332a3 (commit in our private repo)
